examples/js/loaders/GLTFLoader.js

```diff
@@ -2205,6 +2240,27 @@ THREE.GLTFLoader = ( function () {
 
 		} );
 
+		// Meshes can (and should) be reused by multiple nodes in a glTF asset. To
+		// avoid having more than one THREE.Mesh with the same name, count
+		// references and rename instances below.
+		//
+		// Example: CesiumMilkTruck sample model reuses "Wheel" meshes.
+		nodes.forEach( function ( nodeDef ) {
```

Comment:
By the way... better to use `for in` in these situations I think. Using `forEach` in this manner tends to create a lot of garbage.